src/lava/magma/runtime/message_infrastructure/test/cpptest/test_ddschannel.cc

```diff
@@ -70,17 +74,31 @@ void dds_target_fn_a2_bound(int loop,
   }
 }
 
-TEST(TestDDSDelivery, DDSLoop) {
+void dds_protocol(std::string topic_name,
```

Comment:
add printing `loop timedelta` that is same with `TestSocketChannel` and `TestShmDelivery`.